src/Jackett.Common/Indexers/FileList.cs

```diff
@@ -141,25 +143,26 @@ protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuer
             var results = response.Content;
             try
             {
-                CQ dom = results;
-                var globalFreeLeech = dom.Find("div.globalFreeLeech").Any();
-                var rows = dom[".torrentrow"];
+                var parser = new HtmlParser();
+                var dom = parser.ParseDocument(results);
+                var globalFreeLeech = dom.QuerySelectorAll("div.globalFreeLeech").Any();
+                var rows = dom.QuerySelectorAll(".torrentrow");
                 foreach (var row in rows)
                 {
                     var release = new ReleaseInfo();
-                    var qRow = row.Cq();
-                    var qTitleLink = qRow.Find(".torrenttable:eq(1) a").First();
-                    release.Title = qRow.Find(".torrenttable:eq(1) b").Text();
-                    var longtitle = qRow.Find(".torrenttable:eq(1) a[title]").Attr("title");
+                    
+                    var qTitleLink = row.QuerySelectorAll(".torrenttable:eq(1) a").First();
```

Comment:
var qTitleLink = row.QuerySelector(".torrenttable:nth-of-type(2) a");
There are other cases below.